manifest.js

```diff
@@ -2,7 +2,7 @@
 const Confidence = require('confidence');
 const Config = require('./config');
 const Path = require('path');
-
+const Package = require('./package.json');
```

Comment:
Please group all `require`s alphabetically and leave two empty lines after the last one.